tests/integration/examples.py

```diff
@@ -11,19 +12,29 @@
 from testing import run
 
 
+def svstat_stdout_check(service, state):
+    p = Popen(('svstat', 'playground/{}'.format(service)), stdout=PIPE, stderr=PIPE)
+    stdout, stderr = run(p)
+    assert stderr == ''
+    assert 'playground/{0}: {1}'.format(service, state) in stdout
```

Comment:
can we assert == rather than in?

If you need a regex, see testfixtures.StringComparison

https://pythonhosted.org/testfixtures/comparing.html?highlight=stringcomparison#string-comparison-objects